lib/config/options/index.ts

```diff
@@ -1,4 +1,5 @@
 import { getManagers } from '../../modules/manager';
+import { supportedDatasources } from '../../modules/manager/ansible';
```

Comment:
seems to be a wrong import, should be the following instead:
https://github.com/renovatebot/renovate/blob/e035f0562d9ab4772c05116c3b753c820785a41c/lib/config/presets/internal/merge-confidence.ts#L3-L11